src/app/view/components/dot-persona-selector/dot-persona-selector.component.scss

```diff
@@ -0,0 +1,72 @@
+@import "variables";
+@import "mixins";
+
+::ng-deep .dot-persona-selector {
+    margin-top: 0;
+
+    .ui-overlaypanel-content {
```

Comment:
Please move all this to the searchable component, let's take this opportunity to update the UI of the default one:

<img width="762" alt="Screen Shot 2019-07-25 at 10 35 13 AM" src="https://user-images.githubusercontent.com/751424/61891912-f3971300-aec7-11e9-8e9f-1ef247554de9.png">


Zeplin: https://zpl.io/bLWdD7d